CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ document.
 
 [69f99e7...master](https://github.com/rust-lang/rust-clippy/compare/69f99e7...master)
 
+* Move `unneeded_field_pattern` to pedantic group [#1741](https://github.com/rust-lang/rust-clippy/issues/1741)
+
```

Comment:
You don't need to update CHANGELOG in this PR, it's usually updated in release time.

Reply:
sgtm! Updated w/o this change.